src/main/java/com/denizenscript/depenizen/bukkit/events/mythicmobs/MythicMobsDeathEvent.java

```diff
@@ -32,6 +32,8 @@
     //
     // @Regex ^on mythicmob [^\s]+ (dies|death|killed)( by [^\s]+)?( in ((notable (cuboid|ellipsoid))|([^\s]+)))?$
```

Comment:
fix the other meta too (the event lines and the regex still have inline in area)

Reply:
Oof, will do